### PR TITLE
fix tint issue on Lollipop

### PR DESCRIPTION
**Pull request by andela-wakinsanmi**
Add check for pre lollipop to fix the tint issue

### CODE REVIEW

File: layer-atlas/src/main/java/com/layer/ui/composebar/ComposeBar.java [hunk 1]
```diff
@@ -298,7 +299,11 @@ protected void addAttachmentMenuItem(AttachmentSender sender) {
 
         if (sender.getIcon() != null) {
             Drawable iconDrawable = ContextCompat.getDrawable(getContext(), sender.getIcon());
-            DrawableCompat.setTint(iconDrawable, getResources().getColor(R.color.layer_ui_icon_enabled));
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+                DrawableCompat.setTint(iconDrawable, getResources().getColor(R.color.layer_ui_icon_enabled));
```
Comment (smpete) [inline layer-atlas/src/main/java/com/layer/ui/composebar/ComposeBar.java]: Shouldn't DrawableCompat negate the need to do this check? What is wrong with using DrawableCompat?

Comment (andela-wakinsanmi) [reply layer-atlas/src/main/java/com/layer/ui/composebar/ComposeBar.java]: Yes, the problem with that is the setTint doesn't work well with the Drawable on Lollipop. Another approach will be to use a wrapped drawable. 
       Drawable wrappedDrawable = DrawableCompat.wrap(iconDrawable); I

Comment (smpete) [reply layer-atlas/src/main/java/com/layer/ui/composebar/ComposeBar.java]: Why doesn't it work? Do you just need to set the tint mode on the drawable? This is kind of the point of `DrawableCompat` so there is likely an issue elsewhere. Unless you can explain why `DrawableCompat` will not work let's keep investigating this.

Comment (andela-wakinsanmi) [reply layer-atlas/src/main/java/com/layer/ui/composebar/ComposeBar.java]: I mean ```  if (sender.getIcon() != null) {
            Drawable iconDrawable = ContextCompat.getDrawable(getContext(), sender.getIcon());
            Drawable wrappedDrawable = DrawableCompat.wrap(iconDrawable);
            DrawableCompat.setTint(wrappedDrawable, getResources().getColor(R.color.layer_ui_icon_enabled));
            binding.title.setCompoundDrawablesWithIntrinsicBounds(wrappedDrawable, null, null, null);
        }```

**Pull request merged**